OutOfSchool/OutOfSchool.IdentityServer/Views/Account/Email/ChangeEmail.cshtml

```diff
@@ -16,7 +16,18 @@
 
         <h3 class="instruction">@SharedLocalizer["Enter new email message"].Value.ToUpper()</h3>
         <div class="warning">
-            <img src="~/icons/warning.svg" class="warning_icon" alt="warning"><p>@SharedLocalizer["Change email alert"]</p>
+            @if (Model.Submit == "emailMismatch")
```

Comment:
not a call to action right now, but will need to merge you after @RBiyovskiy because he improved the template engine

Reply:
ok, i will wait for the  template engine and then refactor mail messages.